### ISSUE TITLE
PlatformSupport - Thread 1: signal SIGABRT

### ISSUE THREAD

**Issue by rmickeyd** [grpc/grpc-swift#652]
### New Issue Checklist

- [X] Updated grpc-swift to the latest version
- [X] I read the [Contribution Guidelines](https://github.com/grpc/grpc-swift/blob/master/CONTRIBUTING.md)
- [X] I searched for [existing GitHub issues](https://github.com/grpc/grpc-swift/issues)(especially, there are many [resolved issues related to SSL](https://github.com/grpc/grpc-swift/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aclosed+ssl))

### Issue Description

When I try to create a new EventLoopGroup using PlatformSupport.makeEventLoopGroup I get an error within the function at the logging statement of the networkPreference

```
let group = PlatformSupport.makeEventLoopGroup(loopCount: 1)
let configuration = ClientConnection.Configuration(target: ConnectionTarget.hostAndPort("host", port), eventLoopGroup: group)
let clientConnection = ClientConnection(configuration: configuration)
self.service = GRPCServiceClient(connection: clientConnection)
```

<img width="1054" alt="Screen Shot 2019-12-10 at 12 49 14 PM" src="https://user-images.githubusercontent.com/11418552/70567878-9c7a9d80-1b4b-11ea-8b81-85b76ed87f6a.png">

### Environment

<details>
  <pre>
  
| Key                           | Value                                                         |
| ----------------------------- | --------------------------------------------------------------|
| OS Version                    |   10.15                                                      |
| Swift Version                 |    5                                                 |
| Xcode Version                 |   11.2.1                                                       |
| gRPC-Swift Version            |   1.0.0-alpha6                                                       |
| protoc Version                |   3.10.1                                                       |
| protoc-gen-swift Version      |   1.7.0                                                      |
| protoc-gen-grpc-swift Version  |                                                          |
  </pre>
</details>

**Comment by weissi** [comment]
@rmickeyd oh that's interesting. At first sight, this appears to be a problem in [swift-log](https://github.com/apple/swift-log) or the log handler you use. But because [SR-11564](https://bugs.swift.org/browse/SR-11564) let's see if it's that one too. Please apologise all these questions:

- what `LogHandler` (if any) do you use with swift-log?
- does this crash maybe only appear on device and Simulator but _not_ on macOS/Linux? If yes, could you try [this workaround](https://github.com/apple/swift-nio/issues/1294#issuecomment-562367624)? If that fixes it, this is Swift bug [SR-11564](https://bugs.swift.org/browse/SR-11564). 
- could you send us a stack trace (just type `bt all` into the `(lldb)` console in the bottom right in Xcode)

**Comment by rmickeyd** [comment]
This does indeed seem to be related Swift Bug [SR-11564](https://bugs.swift.org/browse/SR-11564l).  Applying [the workaround](https://github.com/apple/swift-nio/issues/1294#issuecomment-562367624) has fixed the issue.  Thanks @weissi!

**Comment by weissi** [comment]
@rmickeyd glad it helped. Out of interest, was this Xcode 11.2 or 11.3?

**Comment by rmickeyd** [comment]
This was Xcode 11.2.1.  I will update to 11.3 now and comment on whether it still happens or not.

**Comment by weissi** [comment]
thank you!

**Comment by rmickeyd** [comment]
The Swift bug seems to have been fixed in Xcode 11.3  No need to apply the workaround anymore.
